tests/acceptance/test_hooked_transformer.py

```diff
@@ -167,6 +167,66 @@ def test_from_pretrained_revision():
         raise AssertionError("Should have raised an error")
 
 
+def check_norm_folding(
+    model_name,
+    hf_model=None,
+    tokenizer=None,
+    prompt="Hello, world!",
+    device=None,
+    dtype=None,
+):
+    """
+    Checks that loading a model with Layer/RMS Norm folding enabled does not (significantly) change its outputs.
+
+    :return: The maximum difference between the logits produced by the same model with and without norm folding enabled.
+    """
+
+    # If a device/dtype is not specified, and hf_model is provided, use its device/dtype
+    # Otherwise, default to cuda (if available)/float32
+    if device is None:
+        if hf_model:
+            device = hf_model.device
+        else:
+            device = "cuda" if torch.cuda.is_available() else "cpu"
+    if dtype is None:
+        if hf_model:
+            dtype = hf_model.dtype
+        else:
+            dtype = "float32"
+
+    folded = HookedTransformer.from_pretrained(
+        model_name=model_name,
+        hf_model=hf_model,
+        device=device,
+        tokenizer=tokenizer,
+        dtype=dtype,
+        fold_ln=True,
+    )
+    tokens = folded.tokenizer.encode(prompt, return_tensors="pt")
```

Comment:
Nit: Can you use folded.to_tokens, not folded.tokenizer.encode? It's the same thing, but I prefer tests to illustrate library best practices